game/scripts/vscripts/TutorialGraph/Steps.ts

```diff
@@ -37,6 +37,35 @@ export const goToLocation = (location: Vector) => {
     })
 }
 
+/**
+ * Creates a tutorial step that spawns and moves a unit.
+ * @param unitName Name of the unit to spawn.
+ * @param spawnLocation Location to spawn the unit at.
+ * @param moveLocation Location to spawn the unit at.
+ * @param team Team the unit belongs to.
+ * @param entityKey Entity key for storing CBaseEntity member in the context.
+ */
+export const spawnAndMoveUnit = (unitName: string, spawnLocation: Vector, moveLocation: Vector, team: DotaTeam, entityKey?: string) => {
+    let unit: CDOTA_BaseNPC
+
+    return step((context, complete) => {
+        unit = CreateUnitByName(unitName, spawnLocation, true, undefined, undefined, team)
```

Comment:
Might be better to use CreateUnitByNameAsync here

Reply:
Didn't know it exists, using it now